src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.scss

```diff
@@ -1,10 +1,15 @@
 @import "scss/constants";
 
-.Container {
+.VenueRoomsEditor {
   &__room-preview {
     position: absolute;
   }
 
+  &__background-image-wrapper {
+    width: 100%;
+    height: 95vh;
```

Comment:
This looks weird, and shouldn't be 95vh. Also the bottom of the map preview is cut off at the bottom.

Reply:
@denisdimitrov in task it was said that  image should be cut proportionately below on preview

Reply:
I understand that but the border radius is being cut off and that's not how it should look like.

Staging:

![stain](https://user-images.githubusercontent.com/18435853/134486325-10b9671a-2a14-467d-8150-b854f2dda336.jpg)

Your branch:
![featurebr](https://user-images.githubusercontent.com/18435853/134486344-f01f5f3d-b846-4ce6-94dc-e364ce213707.jpg)

You are free to 'cut' or change the ratio/dimensions as you like but the styles should be the same. The final result should have border radiuses.

Reply:
@denisdimitrov 
checked and it's ok for me
![image](https://user-images.githubusercontent.com/86356678/134507969-fee5bfd0-ed84-4a2d-910d-2416701110b5.png)

Reply:
It seems to work properly with uploaded map backgrounds but not with our default ones. They are being cut off.

Reply:
And when I open my console and resize it, it's overlapping the map background.

Reply:
I don't have the time right now to test it in more depth, but you should also check if it works properly on all of the common screen resolutions. 1336x768 1920x1080

Reply:
@denisdimitrov fixed in 947c45de5e044303b8d49cda28510851719c37f4